<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1349</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_plot">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Single</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>91</width>
      <height>156</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_init">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Init</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_start">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_stop">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_expert">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Expert</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BRAM</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>284</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="ui_bram0_addr">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="ui_bram0_maplen">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ui_bram0_size">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="txt_nbr_bram">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nb. of BRAM</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="txt_bram0_start_addr">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="txt_bram0_len">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Bram length</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="txt_Onebram_size">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Size per BRAM</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ui_nbr_bram">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ui_bram0_offset">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="txt_bram0_offset">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Data Offset</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="ui_trig_addr">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="txt_trig_addr">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Trigger offset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="msgbox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>345</width>
      <height>24</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QWidget" name="formLayoutWidget_2">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Server IP</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="server_ip">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>172.16.6.85</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_3">
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="server_port">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>50002</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="CustomWidget" name="rtplot" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>941</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_trig">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Trig</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>560</y>
      <width>643</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="bram0">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Bram0 (Q Green)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bram1">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Bram1(Q Blue)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bram2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Bram2(Q Red)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bram3">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Bram3(Q Yelow)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="current_dt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Now:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>600</y>
      <width>611</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="bram0_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Bram0 (I Green)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bram1_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Bram1(I Blue)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bram2_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Bram2(I Red)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bram3_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Bram3(I Yelow)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="bram0_num_data">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>560</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1024</string>
    </property>
   </widget>
   <widget class="QLabel" name="now">
    <property name="geometry">
     <rect>
      <x>1065</x>
      <y>560</y>
      <width>129</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N. of data / line</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>630</y>
      <width>159</width>
      <height>34</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>button_plot</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>label_6</zorder>
   <zorder>formLayoutWidget</zorder>
   <zorder>msgbox</zorder>
   <zorder>splitter</zorder>
   <zorder>rtplot</zorder>
   <zorder>button_trig</zorder>
   <zorder>current_dt</zorder>
   <zorder>label_2</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>bram0_num_data</zorder>
   <zorder>now</zorder>
   <zorder>splitter_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1349</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomWidget</class>
   <extends>QWidget</extends>
   <header location="global">Plotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_stop</sender>
   <signal>clicked()</signal>
   <receiver>button_plot</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_init</sender>
   <signal>clicked()</signal>
   <receiver>button_start</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_expert</sender>
   <signal>clicked()</signal>
   <receiver>ui_bram0_addr</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>508</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_expert</sender>
   <signal>clicked()</signal>
   <receiver>ui_bram0_offset</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_expert</sender>
   <signal>clicked()</signal>
   <receiver>button_trig</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_expert</sender>
   <signal>clicked()</signal>
   <receiver>button_plot</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_stop</sender>
   <signal>clicked()</signal>
   <receiver>button_trig</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_expert</sender>
   <signal>clicked()</signal>
   <receiver>txt_bram0_start_addr</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_expert</sender>
   <signal>clicked()</signal>
   <receiver>txt_bram0_offset</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>547</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_expert</sender>
   <signal>clicked()</signal>
   <receiver>txt_trig_addr</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_expert</sender>
   <signal>clicked()</signal>
   <receiver>txt_bram0_len</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>592</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_expert</sender>
   <signal>clicked()</signal>
   <receiver>ui_bram0_maplen</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>587</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
